Examples/MappedInDemoV2/BasicWithStoryBoard/MapViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  MapViewController.swift
+//  mapBox-Test
+//
+//  Created by Youel Kaisar on 2020-08-27
+//  Copyright © 2020 Mappedin. All rights reserved.
+//
+
+import UIKit
+import Mapbox
+import Mappedin
+
+class MapViewController: UIViewController{
+    
+    var venueSlug: String? = "the-dubai-mall"
+    var venue:MiVenue!
+
+    var mapView: MiMapView!
+
+    @IBOutlet weak var levelUp: UIButton!
+    @IBOutlet weak var levelDown: UIButton!
+    
+    @IBAction func didTapLevelUp(_ sender: UIButton) {
+        incrementLevel()
+    }
+
+    @IBAction func didTapLevelDown(_ sender: UIButton) {
+        decrementLevel()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let mappedIn = Mappedin()
+        let customStyleURL = Bundle.main.url(forResource: "third_party_style", withExtension: "json")!
+        mapView = MiMapView(frame: view.bounds, styleURL: customStyleURL)
+        view.addSubview(mapView)
+        view.sendSubviewToBack(mapView)
+        mapView.miDelegate = self
+        
+        if let venueSlug = self.venueSlug{
+            mappedIn.getVenue(venueSlug: venueSlug, completionHandler: {(resultCode, venue) -> Void in
+                self.venue = venue
+                self.mapView.loadMap(venue: venue!)
+            })
+        }
+    }
+
+    func incrementLevel() {
```

Comment:
do we actually need these? I think that calling mapView.incrementLevel() will do nothing if you can't.

Reply:
You mean the canIncrementLevel?

Reply:
I mean in @IBAction func didTapLevelUp(_ sender: UIButton) {

just use mapView.incrementLevel().

but now I realize we shouldn't even have these buttons, they aren't a part of the quick start guide at all. I think it would be better to just start a blank project and add our SDK in.